client/src/app/editor/file-outputs/file-outputs.component.ts

```diff
@@ -36,17 +39,17 @@ export class OutputFilesDataSource extends DataSource<any> {
 })
 export class FileOutputsComponent implements OnChanges, OnInit {
   @Input() executionId: string;
+  @Input() lab: Lab;
 
-  displayedColumns = ['name', 'created', 'size', 'contentType', 'actions'];
+  outputFiles$: Observable<OutputFile[]>;
+  isLoading$: Observable<boolean>;
+  executionId$ = new ReplaySubject<string>(1);
```

Comment:
ReplaySubject(1) <-- sorry to ask, as I'm not familiar with that API, what does `1` do here?

Reply:
The parameter of a ReplaySubject is the `bufferSize` that says how many items should be replayed upon subscription.

Reply:
Got it, thanks!